File: src/main/java/uk/co/alt236/apkcompare/comparators/results/comparisons/Comparison.java
package uk.co.alt236.apkcompare.comparators.results.comparisons;

import uk.co.alt236.apkcompare.comparators.results.ComparisonResult;

import javax.annotation.Nullable;

public interface Comparison extends ComparisonResult {

    @Nullable
    String getValue1AsString();

    @Nullable
    String getValue2AsString();

    @Nullable
    String getComparedAttribute();

}

File: app/src/main/java/uk/co/alt236/apkcompare/app/comparators/results/comparisons/Comparison.kt
package uk.co.alt236.apkcompare.app.comparators.results.comparisons


import uk.co.alt236.apkcompare.app.comparators.results.ComparisonResult

interface Comparison : ComparisonResult {

    val value1AsString: String?

    val value2AsString: String?

    val comparedAttribute: String?

}
